docs/troubleshooting/detections/detection-rules.asciidoc

```diff
@@ -76,3 +76,40 @@ You can resolve this by expanding the time range, or by configuring {kib}'s auto
 
 CAUTION: Turning off `autocomplete:useTimeRange` could cause performance issues if the data set is especially large.
 ====
+
+[discrete]
+[[rule-exceptions-field-conflicts]]
+.Warning about type conflicts and unmapped fields 
+[%collapsible]
+====
+
+A warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]) and message appear for fields that have <<fields-with-conflicting-types,type conflicts>> or are <<unmapped-field-conflict,unmapped>> across multiple indices. You can learn more about the conflict by hovering over the field. After you select it, the warning message is displayed beneath the field.
```

Comment:
```suggestion
A warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]) and message appear for fields with <<fields-with-conflicting-types,type conflicts>> and <<unmapped-field-conflict,unmapped>> fields. You can learn more about the problem by hovering over the field. After you select it, the warning message appears beneath the field.
```
I could be wrong but I don't think "across multiple indices" applies to unmapped fields, right? They just aren't mapped within a particular index, if I understand correctly.

Reply:
I actually prefer `conflict` vs. `problem` here.

Reply:
@benironside your understanding is correct for unmapped fields (i.e., multiple fields can be unmapped in multiple indices) but "across multiple indices" still applies for type conflicts because a field can be defined as different types in multiple indices. 

Also -- and @e40pud correct me if I'm wrong -- my understanding is the warning appears if one or both of the conditions are met (i.e., when a field has type conflicts, is unmapped, or both) so "or" is more appropriate than "and" in the first sentence. For example:

```suggestion
A warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]) and message appear for fields with <<fields-with-conflicting-types,type conflicts>> across multiple indices or are <<unmapped-field-conflict,unmapped fields>>. You can learn more about the conflict by hovering over the field. After you select it, the warning message appears beneath the field.
```